sdk/python/kfp/__init__.py

```diff
@@ -14,13 +14,14 @@
 
 # `kfp` is a namespace package.
 # https://packaging.python.org/guides/packaging-namespace-packages/#pkgutil-style-namespace-packages
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
```

Comment:
Just for my education: is there a specific reason why double quotes are needed?

Reply:
Simply caused by the use of [black code style](https://black.readthedocs.io/en/stable/the_black_code_style.html#:~:text=Why%20settle%20on%20double%20quotes,fonts%20and%20syntax%20highlighting%20used.)